scripts/asos/iem_scraper_example.py

```diff
@@ -6,6 +6,9 @@
 import time
 import datetime
 
+#allows user to import data into a pandas DataFrame.
+import pandas as pd 
```

Comment:
I'm not necessarily keen on introducing a `pandas` requirement into a script that is meant to run on standard python